src/components/SearchField.tsx

```diff
@@ -87,28 +89,37 @@ const SearchField: React.FC = () => {
   const fromDateInputIncrease = (
     event: React.MouseEvent<HTMLButtonElement, MouseEvent>
   ): void => {
-    if (dateRange.fromYear >= dateRange.toYear) return;
+    if (dateRange.fromYear >= dateRange.toYear) {
+      setIsDataValid(true);
```

Comment:
why do we need to set ti to true first?

Reply:
Here changing a state allows handling a message appropriately (make it appear on a screen when a condition met)

Reply:
Ah, confusing with wrong naming then, check comment below

Reply:
fixed